### PR TITLE
PS-5933: Better MTR terminal support (5.7)

### PR DESCRIPTION
**Pull request by dutow**
* Introducing the MTR_TERM environment variable, which defaults to xterm.
  This variable can be configured to basically any visual terminal,
  and setting it in .profile will result in a configured default setting
  for the user.
* Intrudocing the MTR_LLDB environment variable, to deal with versioned
  lldb executable names in ubuntu.

### CODE REVIEW

**Review by satya-bodapati — COMMENTED** [review]
Not planning to remove --gterm as part of this PR?

**Comment by dutow** [comment]
@satya-bodapati I removed all occurrences of opt_gterm

File: mysql-test/mysql-test-run.pl [hunk 1]
```diff
@@ -1205,7 +1224,6 @@ sub command_line_setup {
              'debug-server'             => \$opt_debug_server,
              'gdb'                      => \$opt_gdb,
              # For using gnome-terminal when using --gdb option
```
Comment (satya-bodapati) [inline mysql-test/mysql-test-run.pl]: Remove this dangling comment as well

**Review by satya-bodapati — APPROVED** [review]
Ok to merge after removing that comment

**Pull request merged**